README.md

```diff
@@ -1,9 +1,13 @@
-# README
+# Native Package Installer 
 
-TODO...
+[![Gem Version](https://badge.fury.io/rb/native-package-installer.svg)](https://badge.fury.io/rb/native-package-installer)
+
+Native Package Installer let you install native packages with `gem install` command.
+
+You can install both native packages and C extensions by one action.
```

Comment:
Could you use the description in gemspec?
https://github.com/ruby-gnome/native-package-installer/blob/master/native-package-installer.gemspec#L28-L34

Reply:
Thanks. done.